docs/config/README.md

```diff
@@ -145,14 +145,25 @@ minio server /data
 
 ### Domain
 
-By default, Minio supports path-style requests that are of the format http://mydomain.com/bucket/object. MINIO_DOMAIN environment variable is used to enable virtual-host-style requests. If the request `Host` header matches with `(.+).mydomain.com` then the matched pattern `$1` is used as bucket and the path is used as object. More information on path-style and virtual-host-style [here](http://docs.aws.amazon.com/AmazonS3/latest/dev/RESTAPI.html)
+By default, Minio supports path-style requests that are of the format http://mydomain.com/bucket/object. `MINIO_DOMAIN` environment variable is used to enable virtual-host-style requests. If the request `Host` header matches with `(.+).mydomain.com` then the matched pattern `$1` is used as bucket and the path is used as object. More information on path-style and virtual-host-style [here](http://docs.aws.amazon.com/AmazonS3/latest/dev/RESTAPI.html)
 Example:
 
 ```sh
 export MINIO_DOMAIN=mydomain.com
 minio server /data
 ```
 
+### Drive Sync
+
+By default, Minio writes to disk in synchronous mode for all metadata operations. Set `MINIO_DRIVE_SYNC` environment variable to enable synchronous mode for all data operations as well.
```

Comment:
It is probably good to add that this flag will prevent having corrupted data after power crash

Reply:
No, it is not necessary to add such things, then we are willfully indicating that we are writing corrupted data. Such is not the case in case of data its an appended data if one of the blocks is missing it is not a problem we have it erasure-coded. This is for rarest of the rarest situations. 

The issue is not because we are writing corrupted data issue is that backend filesystems don't handle power crashes properly. Let's not add such messaging in our docs indicating incorrect things.